Wikibase/Sniffs/Commenting/DisallowedDocTagsSniff.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Custom sniff that reports deprecated PHPDoc tags (e.g. known misspellings) and replaces them with
+ * their preferred counterparts.
+ *
+ * @license GPL-2.0+
+ * @author Thiemo Mättig
+ */
+class Wikibase_Sniffs_Commenting_DisallowedDocTagsSniff implements PHP_CodeSniffer_Sniff {
+
+	/**
+	 * @var array Set a value to null if a tag is forbidden, but no automatic fix should be made.
```

Comment:
What are you missing? The pretty much only way to see this comment is when you look at the code. The defaults are there, directly below. They can be overridden via `<properties>`.

All that should be better documented, I agree. But this is for a later patch.